ssr-ts.js

```diff
@@ -7,9 +7,28 @@
 'use strict';
 
 module.exports = {
-    extends: [
-        require.resolve('./ssr'),
-        // Must be second to override the default parser options
-        require.resolve('./lib/typescript'),
+    extends: [require.resolve('./ssr')],
+    parser: '@typescript-eslint/parser',
+    overrides: [
+        {
+            files: ['**/*.ts'],
+            processor: '@lwc/lwc/ssr',
+        },
+        {
+            files: ['**/*.ssrjs', '**/*.ssrts'],
+            parser: '@typescript-eslint/parser',
```

Comment:
This should be declared as a peer-dependency, if users have to use this config. Also, the remaining typescript configs follow a different approach https://github.com/salesforce/eslint-config-lwc?tab=readme-ov-file#experimental-usage-with-typescript. Can we align all of them to follow one pattern?